test/integration/data/malformed-redactions-json/restored.json

```diff
@@ -0,0 +1,12 @@
+{
+  "basic example": {
```

Comment:
for these examples, could you instead do something like, `malformed redaction`, `missing redaction`, `extra redaction`, `content that should have been redacted`, etc?